src/modules/flow/converter/converter.c

```diff
@@ -1971,4 +1972,63 @@ string_to_blob_convert(struct sol_flow_node *node, void *data, uint16_t port, ui
     return ret;
 }
 
+static int
+timestamp_to_string_convert(struct sol_flow_node *node, void *data, uint16_t port, uint16_t conn_id, const struct sol_flow_packet *packet)
+{
+    time_t in_value;
+    struct tm time_tm;
+    int r;
+    char out_value[128];
+
+    r = sol_flow_packet_get_timestamp(packet, &in_value);
+    SOL_INT_CHECK(r, < 0, r);
+
+    tzset();
+    if (!localtime_r(&in_value, &time_tm))
+        goto timestamp_error;
+
+    r = strftime(out_value, sizeof(out_value), "%Y-%m-%dT%H:%M:%SZ",
```

Comment:
for `strftime()` and `strptime()` there is no issues with incorrect formatting as the argument is fixed to be `struct tm *`.

The problem with `printf()` and `scanf()` is that they take variable arguments (`varargs...`)